app/routes/events/components/Event.css

```diff
@@ -143,3 +143,11 @@
   margin: 0 auto;
   border-bottom: 1px solid #6772e585;
 }
+
+.noRegistrationsText {
+  text-align: center;
+}
+
+.chartContainer {
+  overflow: hidden;
```

Comment:
Personally I find that this works better on mobile with `flex` and `flex-basis column`, so that the labels are centered, but this is very minor

Reply:
I do not quite understand what you mean here. This overflow hidden here, is just to remove the ugly scroll bars that occur because of recharts has really a lot of unnecessary padding. Putting flex on this container would probably not do too much about the labels themselves, since they are nested in child components. Can you elaborate?